### ISSUE TITLE
Change header view image dynamically

### ISSUE THREAD

**Issue by AkshayChordiya** [ManuelPeinado/FadingActionBar#46]
Hi,

Thanks for this excellent library. But I wanna change the image which is shown in header view.

**Comment by medyo** [comment]
All you need to do is to implement the method : headerView with a custom inflated view.
Then you can load images dynamically.

For example : 

```
FadingActionBarHelper helper = new FadingActionBarHelper()
.actionBarBackground(R.drawable.actionbar_bg)
.contentView(getContentView())
.headerView(getHeaderView());
```

GetHeaderView is a private function which returns : 
`View myview = getLayoutInflater().inflate(YOUR_LAYOUR, null,false);`

Then change your image dynamically like that
`ImageView myimage = ((ImageView)myview.findViewById(R.id.of_my_image));
myimage.setImageResource(x);`

**Comment by marsavela** [comment]
Why you guys make the things so complicated... :P 

First of all:

```
FadingActionBarHelper helper = new FadingActionBarHelper()
                .actionBarBackground(R.drawable.ab_background)
                .headerLayout(R.layout.header)
                .contentLayout(R.layout.scrollview);
        setContentView(helper.createView(this));
```

Then:

```
ImageView myimage = ((ImageView)myview.findViewById(R.id.of_my_image));
myimage.setImageResource(x);
```

And the last but not least:

```
helper.initActionBar(this);
```

For me works as expected. ;)
Greetings.

**Comment by medyo** [comment]
My answer was based on the fact that he gonna change the image multi times in the same activity. That's why i recommended him to keep a reference of the root view through inflated header view.

**Comment by marsavela** [comment]
Yeah, but, as I said, I think 

```
myimage.setImageResource(x);
```

works flawlessly. At least for me.

**Comment by AkshayChordiya** [comment]
@medyo @marsavela 
Thanks guys.
I'll try out your method.

**Comment by AkshayChordiya** [comment]
@medyo
@marsavela 
It didn't work for me. Shows default image only.

**Comment by medyo** [comment]
We need more details, please post your activity and layout content.
use http://pastebin.com/ for that

**Comment by afjcjsbx** [comment]
Hi, i have the same problem, 

this is my code http://pastebin.com/zt75deR6

but didn't works for me...

**Comment by AdarshYadav** [comment]
@medyo  Thanks

**Comment by anishpanthi** [comment]
@medyo suffering from same problem. Please help.

**Comment by medyo** [comment]
@anishpanthi have you tried the method written above ?

**Comment by anishpanthi** [comment]
@medyo Yeah I tried the way that you have explained but it doesn't work for me. Worked in this way:

ImageView headerImage = (ImageView) findViewById(R.id.image_header);
headerImage.setImageResource(academicListImages.getResourceId(position,-1));

rather then using LayoutInflater.

by the way, thankx.

**Comment by medyo** [comment]
I used LayoutInflater to demonstrate how to handle a parent view (with subviews), you can get inspired and do it only for one view like an ImageView with a simple findViewById.

Glad it worked for you.

**Comment by anishpanthi** [comment]
@medyo Thankx dude.
